replicator/src/test/scala/com/evolutiongaming/kafka/journal/replicator/DelayedTopicCommitTest.scala

```diff
@@ -8,15 +8,17 @@ import com.evolutiongaming.skafka.{Offset, Partition}
 import org.scalatest.funsuite.AsyncFunSuite
 import org.scalatest.matchers.should.Matchers
 
+import scala.annotation.nowarn
 import scala.concurrent.duration.*
 
-class TopicCommitTest extends AsyncFunSuite with Matchers {
+@nowarn("cat=deprecation")
```

Comment:
why we need this?

Reply:
`delayed` is marked as deprecated, this whole test lights up with warnings if not suppressed.